File: StudentCredit.Application/Students/Dtos/StudentEducationDto.cs
using StudentCredit.Data.Rdpzsd.Enums;

namespace StudentCredit.Application.Students.Dtos
{
    public class StudentEducationDto : EducationBasicDto
    {
        public CourseType Course { get; set; }
        public string StartPeriod { get; set; }
        public string FinishPeriod { get; set; }
    }
}

File: WebApplication/src/modules/student/models/student-education.dto.ts
import { CourseType } from "src/modules/application/application-one/enums/course-type.enum";
import { EducationBasicDto } from "./education-basic.dto";

export class StudentEducationDto extends EducationBasicDto {
  course: CourseType;
  startPeriod: string;
  finishPeriod: string;
}